### DISCUSSION TITLE
Excluding some services

### DISCUSSION THREAD

**Discussion by ngcheeyuan** [Luctia/ezarr#55] [Q&A] [answered]
Hi, may I know if I can exclude some of the services within the docker-compose.yml  if I already have them on my system? e.g.  I already have qbit and jellyfin running on my server. Or do I need to have a fresh install?

**Comment by Luctia** [comment, marked as answer]
You don't need to create new services, but in this case you should ensure that the different services have access to the existing files. For example, for sonarr, you should make sure that sonarr and jellyfin use the same directory for series. This might also take some effort due to permissions, but if you're comfortable with that, it should be no problem.

**Reply by ngcheeyuan** [reply]
Thanks for the quick reply
